### ISSUE TITLE
Scanner node needs to delete atomic scanner logs after moving those to NFS

### ISSUE THREAD

**Issue by dharmit** [CentOS/container-pipeline-service#176]
`/var/lib/atomic` directory fills up with scan data. This scan data is of no use once data is pushed to NFS:

```
$ du -sh *
3.4M misc-package-updates 
2.4G pipeline-scanner
7.9G rpm-verify
```

Scan worker needs to remove this data to ensure we don't end up with scan node running out of disk space.

**Comment by navidshaikh** [comment]
@dharmit : We need to decide what to delete. Say multiple images' scanning is going on, we can't just delete other scanning results. What about deleting all the results generated 2 days before? Thus we'll have results of scanners for 2 days (always).

**Comment by dharmit** [comment]
> Say multiple images' scanning is going on, we can't just delete other scanning results.

When a scan worker is done executing all scanners on a image, we store the logs onto the NFS. Once this step is over, I'm guessing it's safe to delete logs from scanner node. As long as NFS is backed up, we shouldn't be worried about keeping data for 2 days.

**Comment by navidshaikh** [comment]
>When a scan worker is done executing all scanners on an image, we store the logs onto the NFS. Once this step is over, I'm guessing it's safe to delete logs from scanner node. As long as NFS is backed up, we shouldn't be worried about keeping data for 2 days.

So, if copy to NFS operations completes, we remove the logs from `/var/lib/atomic`, if copy to NFS fails, we keep the logs in `/var/lib/atomic`. We log about the path of scanners results in dev logs. Sounds good ?

**Comment by dharmit** [comment]
> if copy to NFS fails, we keep the logs in /var/lib/atomic

Yes, keep logs and since copy to NFS failed, scan worker should notify some human(s) to look into it. Also, if we decide to keep logs on disk till NFS issue is solved, we need to figure out how to send them to NFS when it's back up.

>  We log about the path of scanners results in dev logs.

Didn't get this one.

**Comment by navidshaikh** [comment]
> scan worker should notify some human(s) to look into it. 

thats what I meant when I mentioned "We log about the path of scanners results in dev logs."

If write to NFS is failing, we keep the logs intact under /var/lib/atomic and in scanner worker logs, we log the issue along with path of scanning results under /var/lib/atomic directory. Devs debugging the issue should be able to see issue and actual logs by looking at scanner worker logs.

**Comment by navidshaikh** [comment]
to be fixed by #558
